aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -5,60 +5,111 @@ description: Learn about configuring endpoints with Kestrel, the cross-platform
 monikerRange: '>= aspnetcore-5.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 03/10/2023
+ms.date: 06/15/2023
 uid: fundamentals/servers/kestrel/endpoints
 ---
 
 # Configure endpoints for the ASP.NET Core Kestrel web server
 
-[!INCLUDE[](~/includes/not-latest-version.md)]
+:::moniker range="< aspnetcore-7.0"
+[!INCLUDE [](~/includes/not-latest-version.md)]
+:::moniker-end
 
 :::moniker range=">= aspnetcore-8.0"
 
+Endpoints in ASP.NET Core Kestrel provide the infrastructure for listening to incoming requests and routing them to the appropriate middleware. An endpoint in Kestrel is defined by a combination of address and protocol.
+
+* The address specifies the network interface and port on which the server listens for incoming requests. This can be a specific IP address, such as localhost or a public IP, along with a port number.
```

Comment:
```suggestion
* The address specifies the network interface and port on which the server listens for incoming requests. This can be a specific IP address with or without a port number. If the port isn't specified, a default port is used, typically 80 for HTTP or 443 for HTTPS. The address can also be a host name, such as `localhost`, which resolves to one or more IP addresses.
```